src/main/resources/assets/spookytime/sounds.json

```diff
@@ -0,0 +1,9 @@
+{
+  "doot": {
+    "category": "players",
+    "sounds": [
+      "spookytime:doot"
+    ],
+    "subtitle": "subtitle.spookytime.doot"
```

Comment:
You set the subtitle translation key here, but then never translated it in the `en_us.json`